e2e-tests/src/test/java/com/wixpress/common/petri/e2e/FilterParametersConfigTest.java

```diff
@@ -0,0 +1,56 @@
+package com.wixpress.common.petri.e2e;
+
+import com.wixpress.petri.test.TestBuilders;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class FilterParametersConfigTest extends BaseTest {
+
+
+    private static Path original;
+    private static Path tempFile;
+
+    @BeforeClass
+    public static void startServers() throws Exception {
```

Comment:
but the SampleAppRunner changes the labProps file before the server starts too..
see [here](https://github.com/wix/petri/blob/4199a995da18a86f566d1199433feefb9329cc07/petri-test-utils/src/main/java/com/wixpress/petri/test/SampleAppRunner.java#L57)

this is in the c'tor, and the start method of SampleAppRunner is called later on.

Am i missing something?